galasa-parent/galasa-boot/build.gradle

```diff
@@ -0,0 +1,58 @@
+plugins {
+    id 'biz.aQute.bnd.builder' version '5.1.2'
```

Comment:
Not required,   boot is not an OSGi project as it starts up OSGi

Reply:
Corrected.